src/v3/src/components/ConsentHeader/ConsentHeader.tsx

```diff
@@ -119,12 +105,11 @@ const ConsentHeader: FunctionComponent = () => {
             textAlign="center"
           >
             <Typography
-              component="h1"
```

Comment:
a11y issue; since admin consent view has a group heading, (which is h3), all heading elements should ascend in order without skipping.

Reply:
While we're here.... could we make the visual presentation of the app name Heading 6?

(But not violate the semantic meaning for a11y.)

Reply:
Done! 
<img width="760" alt="image" src="https://github.com/okta/okta-signin-widget/assets/97472729/4cf8f409-ab88-4275-b2eb-a6f9edb5e17c">